### DISCUSSION TITLE
Is there a way to set ExecutionPolicy Unrestricted only for the GUI.bat each time you use it?

### DISCUSSION THREAD

**Discussion by poweradegator** [bmaltais/kohya_ss#333] [Q&A]
After installing, the GUI and everything, can you turn ExecutionPolicy back to restricted and then somehow only set ExecutionPolicy Unrestricted for the GUI.bat each time you run it to train a LORA?

for Kohya non-GUI script version something like powershell.exe -noprofile -executionpolicy bypass -file .\train.ps1 

is something like that possible for the kohya GUI version?

**Comment by Hany-mohsen-elhassany** [comment]
yeah, i am really curious about this question, setting a policy to Unrestricted all the time is not a very good option.
i also if i made the policy to RemoteSigned and then use `Unblock-File ,` does this work ??

( example7 : https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-7.3 )

**Comment by bmaltais** [comment]
No idea. I just went for the jugular and lifted all blocks... But if you find a way to make the restriction better I will be happy to add it to the instructions.
